docs/install/2-install-server-with-docker.rst

```diff
@@ -181,39 +195,11 @@ Overwrite default server configuration
 ######################################
 
 By default the server will use the file located in the image at ``/etc/inmanta/inmanta.cfg``.
-If you want to change it, you can copy this file, edit it, then mount it in the container,
+If you want to change it, the recommended way is to provide the server config options via environment variables as done in the above example.
+All the different options and associated env variables are described :ref:`here<config_reference>`.
+It is also possible to provide the entire file. You can copy and edit it, then mount it in the container,
 where the original file was located.
 
-If you use docker-compose, you can simply update this section of the example above:
-
-
-.. only:: iso
-
-    .. code-block:: yaml
-        :substitutions:
-
-        inmanta-server:
-            container_name: inmanta_orchestrator
-            image: containers.inmanta.com/containers/service-orchestrator:|version_major|
-            ports:
-                - 8888:8888
-            volumes:
-                - ./resources/com.inmanta.license:/etc/inmanta/license/com.inmanta.license
-                - ./resources/com.inmanta.jwe:/etc/inmanta/license/com.inmanta.jwe
-                - ./resources/my-server-conf.cfg:/etc/inmanta/inmanta.cfg
-
-.. only:: oss
-
-    .. code-block:: yaml
-
-        inmanta-server:
-            container_name: inmanta_orchestrator
-            image: ghcr.io/inmanta/orchestrator:latest
-            ports:
-                - 8888:8888
-            volumes:
-                - ./resources/my-server-conf.cfg:/etc/inmanta/inmanta.cfg
-
 
 Starting the ssh server
```

Comment:
This section will be addressed in this ticket: https://github.com/inmanta/inmanta-core/issues/8421